

arch/x86_64/kernel/genapic.c:23: warning: type defaults to `int' in declaration of `EXPORT_SYMBOL'
arch/x86_64/kernel/genapic.c:23: warning: parameter names (without types) in function declaration
arch/x86_64/kernel/genapic.c:23: warning: data definition has no type or storage class
arch/x86_64/kernel/genapic_cluster.c:61: warning: `cluster_cpu_present_to_apicid' defined but not used

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/x86_64/kernel/genapic.c |    2 ++
 1 files changed, 2 insertions(+)

diff -puN arch/x86_64/kernel/genapic.c~x86-64-clustered-apic-support-fix-fix arch/x86_64/kernel/genapic.c
--- 25/arch/x86_64/kernel/genapic.c~x86-64-clustered-apic-support-fix-fix	Thu Oct  7 14:22:46 2004
+++ 25-akpm/arch/x86_64/kernel/genapic.c	Thu Oct  7 14:22:46 2004
@@ -15,6 +15,8 @@
 #include <linux/kernel.h>
 #include <linux/ctype.h>
 #include <linux/init.h>
+#include <linux/module.h>
+
 #include <asm/smp.h>
 #include <asm/ipi.h>
 
_
